compiler/src/iree/compiler/Codegen/LLVMCPU/test/select_x86_64_lowering_strategy.mlir

```diff
@@ -1943,3 +1943,41 @@ module {
 //      CHECK:   iree_linalg_ext.custom_op
 // CHECK-SAME:       lowering_config = #[[CONFIG]]
 //  CHECK-NOT:   lowering_config
+
+// -----
+
+#executable_target = #hal.executable.target<"llvm-cpu", "embedded-elf-x86_64",
+    {cpu_features = "+avx512f",
+    data_layout = "e-m:e-p270:32:32-p271:32:32-p272:64:64-i64:64-f80:128-n8:16:32:64-S128",
+    native_vector_size = 16 : index, target_triple = "x86_64-none-elf"}>
+func.func @test_tiling_cpu_default() attributes {hal.executable.target = #executable_target} {
```

Comment:
Sorry, I think I made a bad suggestion in terms of the tests. This is a little fragile to me because it could go with other pipelines once we support vectorization for this type of conv ops. To get a robust (and simpler IR) test, we can use a linalg.generic op with non-trivial indexing maps or just a linalg.quantized_matmul op. Also, we don't need the consumer op (i.e., linalg.generic ... ) in this case because it could be difficult to maintain the tests. We should try to minimize the test case, see https://llvm.org/docs/TestingGuide.html#best-practices-for-regression-tests for details.

(I'm okay with linalg.quantized_matmul op because we always decompose it at very beginning and I don't see a plan to support the op codegen directly. It is a more robust case to me.)

Reply:
Wait, doesn't `linalg.quantized_matmul` decompose to `linalg.matmul` and have no dispatch going down the CPUDefault path? Here's the ir dump https://gist.github.com/pashu123/6a5fd693bb58aac8e1c6e9edd9d3386e

Reply:
Yes, you're right. That's why I suggested to use it for CPUDefault path. Because we don't expect a non-decomposed linalg.quantized_matmul being an input by default. It's kind of a hack, but it is more stable to me. (note: we can add a comment to the test case saying that it is mainly for testing CPUDefault pipeline.)

Reply:
Thanks, it makes a lot more sense now.